### ISSUE TITLE
Issue with k8s.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/

### ISSUE THREAD

**Issue by khteh** [kubernetes/website#16479]
**This is a Bug Report**

**Problem:**
How to use:
```
type: Object
object:
  metric:
    name: requests-per-second
  describedObject:
    apiVersion: networking.k8s.io/v1beta1
    kind: Ingress
    name: main-route
  target:
    type: Value
    value: 2k
```
**Proposed Solution:**
I added it in my hpa but gets the following:
```
frontend-hpa          StatefulSet/biz4x-frontend    <unknown>/10k, <unknown>/75%   2         5         2          58d
```
**Page to Update:**
https://kubernetes.io/...

1.13

**Comment by sftim** [comment]
@khteh I don't think I understand this report.

I'm not sure if you have spotted a mistake in the documentation, or if you have found it hard to follow and want help. Can you clarify?
/triage needs-information

**Comment by khteh** [comment]
Exactly what I described in my original post. The documentation presents custom metrics, `Object` in this case but doesn't show the solution which collects the object and send it to the HPA.
